stream.go

```diff
@@ -45,11 +45,12 @@ type Channel int
 // Event represents a MIDI event.
 type Event struct {
```

Comment:
Let's keep the number of types small.

type Event struct {
    //...
   Message []byte
}